integration/sql/iface-java/src/main/java/io/openlineage/sql/SqlMeta.java

```diff
@@ -13,11 +13,13 @@ public class SqlMeta {
   private final List<DbTableMeta> inTables;
   private final List<DbTableMeta> outTables;
   private final List<ColumnLineage> columnLineage;
+  private final List<ExtractionError> errors;
 
-  public SqlMeta(List<DbTableMeta> in, List<DbTableMeta> out, List<ColumnLineage> columnLineage) {
+  public SqlMeta(List<DbTableMeta> in, List<DbTableMeta> out, List<ColumnLineage> columnLineage, List<ExtractionError> errors) {
```

Comment:
Wouldn't it be possible to throw Java exception `SqlParserException` instead of returning valid `SqlMeta` with error fields?

Reply:
The point of this task is to allow for partial success. Look at `returnedMultipleErrors` test.

Reply:
Agreed on passing those exceptions through. Same happens in Python.

Reply:
> passing exceptions through
With you @mobuchowski :)